.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/node:4-browsers
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run: npm config set "//registry.npmjs.org/:_authToken" $NPM_AUTH
+      - run: make test
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - node_modules
+    publish:
+      docker:
+        - image: circleci/node:4-browsers
+      steps:
+        - attach_workspace: { at: . }
+        - run: npm publish .
+workflows:
+  version: 2
+  test_and_publish:
+    jobs:
+      - test:
+          filters:
+            tags:
+              only: /.*/
+      - publish:
+          requires:
+            - test
+          filters:
+            tags:
+              only: /[0-9]+(\.[0-9]+)*(-.+)?/
```

Comment:
@ccnixon @Peripheral1994 would love to get some on eyes on this part to ensure we're only deploying under two conditions (as before):

1. branch is master
2. version is tagged